test/Runner.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Runner } from "../src/Runner";
+
+describe("Runner", () => {
+  class CustomRunner extends Runner {
+    public stdout: string = "";
+    public stderr: string = "";
+    public exitCode: number;
```

Comment:
I think `exitCode` should default to `0`. Silly mistake on my part. Appreciate if you could fix :).